lua/lazyvim/plugins/ui.lua

```diff
@@ -308,7 +265,26 @@ return {
   { "MunifTanjim/nui.nvim", lazy = true },
 
   {
-    "folke/snacks.nvim",
+    "snacks.nvim",
```

Comment:
Noticed you have these as `"folke/snacks.nvim"` on `init.lua`, but dropped the `"folke"` here.

(P.S. how does that work?)

Reply:
Yes, that works. As long as it's specified in full somewhere.

Reply:
TIL! Thanks!